.github/MODEL_CARD.md

```diff
@@ -162,13 +179,24 @@ The model can be deployed on any Linux service or cloud environment. Currently,
 
 #### Hardware
 
-AWS EC2 Instances
 
+The model can be deployed on any Linux service or cloud environment. Currently, the model and infrastructure are trained and deployed on AWS using services including Lambda, S3, Athena, EC2, and EBS.
+
+#### Hardware
+
+AWS EC2 Instances:
+
+- **Instance Type:** m5.xlarge
+- **RAM:** 16 GB
+- **SSD:** 100 GB
+- **vCPUs:** 4
+
+These specifications provide a balance of computational power, memory, and storage to efficiently handle the training and deployment of the model. The use of AWS EC2 instances ensures scalability and flexibility, allowing for adjustments based on workload requirements.
 #### Software
 
 The model uses Python libraries such as scikit-learn, pandas, and numpy for data processing and model training.
 
-## Citation [optional]
+## Citation
```

Comment:
Is not required to have the head as no details are added.

Reply:
there are details there
```

## Citation

```bibtex
@INPROCEEDINGS{2022AGUFMIN42C0346L,
       author = {{Last}, Christina and {Pramanik}, Prithviraj},
        title = "{Using Open Data to Understand Air Pollution Exposure in Resource-Constrained Environments}",
    booktitle = {AGU Fall Meeting Abstracts},
         year = 2022,
       volume = {2022},
        month = dec,
          eid = {IN42C-0346},
        pages = {IN42C-0346},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2022AGUFMIN42C0346L},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
```bibtex
@article{pramanik2023remote,
  title={Remote Sensing and Machine Learning based models to understand Air Pollution Exposure in Resource-Constrained Environments},
  author={Pramanik, Prithviraj and Last, Christina},
  journal={AGU23},
  year={2023},
  publisher={AGU}
}
```

Reply:
there are details there
```

## Citation

```bibtex
@INPROCEEDINGS{2022AGUFMIN42C0346L,
       author = {{Last}, Christina and {Pramanik}, Prithviraj},
        title = "{Using Open Data to Understand Air Pollution Exposure in Resource-Constrained Environments}",
    booktitle = {AGU Fall Meeting Abstracts},
         year = 2022,
       volume = {2022},
        month = dec,
          eid = {IN42C-0346},
        pages = {IN42C-0346},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2022AGUFMIN42C0346L},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
```bibtex
@article{pramanik2023remote,
  title={Remote Sensing and Machine Learning based models to understand Air Pollution Exposure in Resource-Constrained Environments},
  author={Pramanik, Prithviraj and Last, Christina},
  journal={AGU23},
  year={2023},
  publisher={AGU}
}
```